.github/workflows/specs-model-test.yaml

```diff
@@ -0,0 +1,27 @@
+name: Specs Model - Test
+
+on:
+  push:
+    branches:
+      - main
+      - typespec-next
+      - users/kojamroz/specs_model_1
```

Comment:
FYI The line `- users/kojamroz/specs_model_1` to be removed before merging. it is here for testing.

Reply:
FYI, you should be able to test a new Action by just opening a fork pull request.  Unlike DevOps pipelines, you don't need to create a branch in the main fork.

Reply:
I am unsure how to do that. I do hope I won't have to fiddle with forks to test my action. I added the ability to manually trigger the workflow.